scripts/start_krun_from_cron

```diff
@@ -2,50 +2,42 @@
 #
 # This script is designed to be invoked as cron @reboot task and as a normal
 # user.
-#
-# The following variables must be in the envrionment:
-#
-# * EXPERIMENT_DIR: Path where your experiment lives, to change to before
-#                   invoking Krun.
-#
-# * CONFIG_FILE: The name of the krun config file, absolute or relative to
-#                EXPERIMENT_DIR.
-#
-# * KRUN_DIR: The directory where Krun lives, absolute or relative to
-#             EXPERIMENT_DIR.
-#
-# The contents of the (optional) EXTRA_KRUN_ARGS environment is appended to the
-# Krun invocation. This is useful for (e.g.) turning on debug flags.
 
-# Find Python
-if [ "`uname`" = "OpenBSD" ]; then
-    PATH=${PATH}:/usr/local/bin
+if [ $# -lt 1 ]; then
+    echo "start_krun_from_cron <absolute-path-to-config-file> [extra krun args]"
+    exit 1
 fi
-PYTHON=`which python2.7`
 
-if [ "${PYTHON}" = "" ]; then
-    echo "Failed to find python"
+# Consume arguments
+SCRIPT_PATH=`readlink -f $0`
```

Comment:
Not that this is a big deal, but `readlink -f` doesn't exist on e.g. OS X. There are, I think, alternative mechanisms (but I don't remember what they are).

Reply:
Doh. I spent some time trying to find something portable. The best I found was `if [[ "$DIR" = /* ]]; then` as mentioned here: https://stackoverflow.com/questions/20204820/check-if-shell-script-1-is-absolute-or-relative-path

Sadly this doesn't seem to work on dash, so the readlink -f think is something i concocted myself.

We can add a hack later if we decide to support OS X (hah!).

Reply:
OK, so what we need to do is something like `SCRIPT_PATH='readlink -f $0' || exit $?` or `set -e` so that if someone does try this on OS X it dies ASAP.